.travis.yml

```diff
@@ -15,23 +15,17 @@ addons:
 
 matrix:
   include:
-    - env: CFLAGS="-O2"
-    - env: CFLAGS="-O2" ABI=32
-      addons:
-        apt:
-          packages:
-          - libgmp-dev:i386
-          - libreadline-dev:i386
-          - zlib1g-dev:i386
-          - gcc-multilib
-          - g++-multilib
+    - env: GAPBRANCH=master
+    - env: GAPBRANCH=stable-4.9
+    - env: GAPBRANCH=stable-4.10
 
 before_script:
-  - export GAPROOT="$HOME/gap"
   - git clone https://github.com/gap-system/pkg-ci-scripts.git scripts
   - scripts/build_gap.sh
 script:
-  - scripts/build_pkg.sh && scripts/run_tests.sh
+  - scripts/build_pkg.sh
+  - $HOME/gap/bin/gap.sh -l "$PWD/gaproot;" --quitonbreak makedoc.g
```

Comment:
`makedoc.g` no longer quits GAP when it finishes. I'm not sure of a nice way of making GAP still quit, but this is one way:
```
bash echo "quit;" | $HOME/gap/bin/gap.sh -l "$PWD/gaproot;" --quitonbreak makedoc.g
```